lab/style-script/gss-graphstyle-map-directive.md

```diff
@@ -0,0 +1,17 @@
+---
+id: gss-graphstyle-map-directive
+title: Graph Style Script @GraphStyle.Map directive properties
+sidebar_label: GraphStyle.Map directive properties
+---
+
```

Comment:
This is a new page? can we have a little intro like the definition you stated in the reference-guide?